src/components/Atoms/Button/Button.js

```diff
@@ -19,10 +22,14 @@ const StyledButton = styled.button`
   align-items: center;
   border: none;
   cursor: pointer;
+  color: ${({ backgroundColor }) => getContrastColor(backgroundColor)};
+  border: 1px solid ${({ backgroundColor }) => getContrastColor(backgroundColor)};
```

Comment:
Why are we adding a border here?

Reply:
Ahh, i added that border while testing it out , will remove that